src/IO/Config.hs

```diff
@@ -39,7 +40,19 @@ defaultConfig = Config {
 }
 
 getDir :: IO FilePath
-getDir = (++ "/.taskell") <$> getHomeDirectory
+getDir =
+    let
```

Comment:
Then get rid of all of these bindings except for `directoryName`. As a general principle, you don't want to `let`-define monadic values that you are only going to use once.